wiki/Ranking_criteria/osu!mania/en.md

```diff
@@ -19,8 +19,13 @@ This set of **osu!mania ranking criteria** lays out [rules and guidelines](/wiki
 ### Common terms
 
 - **BPM**: Tempo of a song measured in beats per minute.
-- **Column**: Vertical lines on the playfield where notes are placed. The editor uses `0|1|2|3|4|5|6|7|8|9` for timestamp labels.
+- **Column**: Vertical lanes of the [playfield](/wiki/Game_mode/osu!mania#playfield) where notes are placed. Note that [editor timestamps](/wiki/Modding/Timestamp) refer to columns by their zero-indexed position starting from the left.
 - **Key mode**: The number of columns present in a difficulty.
+- **Playstyle**: A mapper's intended style of [key bindings](/wiki/Game_mode/osu!mania#controls) or hand positions used to play a difficulty.
+- **Standard playstyle**: The most common type of playstyle for key counts 10 or below, where no column is of particular significance, and the player is expected to control each key with a unique finger.
```

Comment:
`Key modes` and `Standard playstyles` would end up meaning the same thing. Not technically on paper, but with how key modes are understood by the community, it would have that overlap.

Ideally, `Key modes` by definition should already encompass the definition of `Standard playstyles`.

Reply:
they're different things, how I intended it anyway. "key mode" is just the number of columns. "playstyle" is the way that the mapper probably intends you to play them

so the 8K keymode for example may be played with "8K" playstyle (standard), "7K1S" playstyle, etc... that's how I intended to write it anyway.

Reply:
Yep I'm aware of the change. I'm moreso concerned at the way Key mode is perceived by the community. If a player was to see a beatmap listed as 7K, 4K, 5K, etc... they'd assume it is going to be played in the "default way"; in the way a Standard Playstyle is defined.

Would probably make more sense to specify that in Key mode outright.

Reply:
I think the usage of these terms should be more clear now. "standard playstyle" is only referred to once, specifically to distinguish from all playstyles, and now "key mode" and "playstyle" often appear alongside each other to help with readability for people who arent familiar (and don't *need* to be familiar) with any of this playstyle stuff.